src/cmd/version.go

```diff
@@ -18,24 +18,33 @@ import (
 
 	"github.com/zarf-dev/zarf/src/config"
 	"github.com/zarf-dev/zarf/src/config/lang"
-	"github.com/zarf-dev/zarf/src/pkg/message"
 )
 
 type versionOptions struct {
 	outputFormat outputFormat
 	outputWriter io.Writer
 }
 
-func newVersionOptions() *versionOptions {
+func newVersionOptions(writer ...io.Writer) (*versionOptions, error) {
+	if len(writer) > 1 {
```

Comment:
nit: IMO it would read cleaner if we accepted a single writer and then checked if it was nil

Reply:
Fair, I hear what you're saying. I think I agree and I'll add that my choice to use a variadic function here is overly clever. My intent was adding an optional writer arg without breaking callers' expectations of a default writer when calling `newVersionOptions()` with empty args, but that's not actually an issue for how `newVersionOptions` is used.

Looking into it more, there is bug here until we check for nil like you're saying. Simplified the fn with a nil->default in latest revision.